src/Components/App/App.js

```diff
@@ -82,28 +88,29 @@ const App = () => {
        console.log(error, "fetch")
       }
     }
-      
-
-      
-      useEffect(() => {
-        fetchData()
-  }
             
   useEffect(() => {
     fetchData()
 
     socket.on('receive_data', (data) => {
       setVendors([data.truck, ...data.updatedVendors])
+      setPushNote([...pushNote, { vendorName: data.truck.name }])
```

Comment:
SO excited to see this in practice. Thanks for doing all the hard work to get this implemented!